.gitignore

```diff
@@ -2,3 +2,4 @@ test_integration
 node_modules/
 package-lock.json
 .env
+database
```

Comment:
Why ? You should provide SQL requests to build your database
Moreover, since you forgot "/" at the endline, your updates to SQL files were added to your commits